pyomo/repn/plugins/nl_writer.py

```diff
@@ -264,6 +327,15 @@ def __call__(self, model, filename, solver_capability, io_options):
         col_fname = filename_base + '.col'
 
         config = self.config(io_options)
+
+        # There is no (convenient) way to pass the scaling factors or
+        # information about presolved variables back to the solver
+        # through the old "call" interface (since solvers that used that
+        # interface predated scaling / presolve).  We will play it safe
+        # and disable scaling / presolve when called through this API
+        config.scale_model = False
+        config.linear_presolve = False
```

Comment:
Should this issue a warning if they were `True`?

Reply:
Ahh, good point. Maybe we can deprecate the `__call__` after the solver refactor is complete.

Reply:
That is my hope.